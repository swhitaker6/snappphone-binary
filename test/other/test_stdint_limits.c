#include <stdio.h>
#include <stdint.h>

int main () {
  printf("PTRDIFF_MIN: %ti\n", PTRDIFF_MIN);
  printf("PTRDIFF_MAX: %ti\n", PTRDIFF_MAX);

  printf("INTPTR_MIN: %ld\n", INTPTR_MIN);
  printf("INTPTR_MAX: %ld\n", INTPTR_MAX);

  printf("UINTPTR_MAX: %lu\n", UINTPTR_MAX);

  printf("SIZE_MAX: %zu\n", SIZE_MAX);
  return 0;
}
