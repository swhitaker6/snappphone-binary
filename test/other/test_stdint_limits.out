PTRDIFF_MIN: -2147483648
PTRDIFF_MAX: 2147483647
INTPTR_MIN: -2147483648
INTPTR_MAX: 2147483647
UINTPTR_MAX: 4294967295
SIZE_MAX: 4294967295
